src/main/kotlin/no/nav/familie/ef/sak/api/beregning/BeregningRequest.kt

```diff
@@ -6,7 +6,7 @@ import java.time.YearMonth
 
 data class ÅrMånedPeriode(val årMånedFra: YearMonth, val årMånedTil: YearMonth)
```

Comment:
Når denne likevel skal tas i bruk nå, er det tilsrekkelig å bare kalle verdiene `fra` og `til`? 
Contexten er ju "ÅrMånedPeriode"

Reply:
Da må vi isåfall huske å oppdatere det i frontend.